test/testUtils.js

```diff
@@ -102,8 +100,10 @@ function takeScreenshot() {
 
 /* set tracker blocking and https state */
 function resetSettings(settingState) {
-    bkg.settings.updateSetting('trackerBlockingEnabled', settingState)
-    bkg.settings.updateSetting('httpsEverywhereEnabled', settingState)
+    bkg.settings.ready().then(() => {
+        bkg.settings.updateSetting('trackerBlockingEnabled', settingState)
+        bkg.settings.updateSetting('httpsEverywhereEnabled', settingState)
+    })
```

Comment:
I was seeing this error out when I ran the tests after my changes, not sure why it wasn't before.